client/flutter/pubspec.lock

```diff
@@ -65,9 +65,11 @@ packages:
     source: hosted
     version: "0.16.1"
   firebase_analytics:
+  firebase_messaging:
     dependency: "direct main"
     description:
       name: firebase_analytics
+      name: firebase_messaging
```

Comment:
running `flutter pub get` on a fresh checkout of this branch throws the error `Error on line 72, column 7 of pubspec.lock: Duplicate mapping key.`